<ui version="4.0">
 <class>MenuVisitante</class>
 <widget class="QMainWindow" name="MenuVisitante">
    <property name="geometry">
     <rect><x>0</x><y>0</y><width>900</width><height>600</height></rect>
    </property>
    <property name="windowTitle">
     <string>MenULE - Menú Visitante</string>
    </property>
    <property name="windowState">
     <set>Qt::WindowFullScreen</set>
    </property>
    <property name="styleSheet">
     <string>
QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                                            stop:0 #ff9800, stop:1 #ffc107);
}
QLabel#labelUsuario {
        font-size: 20px;
        color: white;
        font-weight: bold;
        padding: 8px;
}
QCalendarWidget {
        background-color: white;
        border-radius: 8px;
}
QTextEdit {
        background-color: #e3f2fd;
        color: #0d47a1;
        border: 1px solid #90caf9;
        border-radius: 6px;
        padding: 10px;
}
QPushButton {
        background-color: #ff9800;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 6px;
}
QPushButton:disabled {
        background-color: #90a4ae;
}
     </string>
    </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QVBoxLayout" name="mainVerticalLayout">
        <item>
         <widget class="QLabel" name="labelUsuario">
            <property name="text">
             <string>Bienvenido, Usuario</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="contentLayout">
            <item>
             <widget class="QCalendarWidget" name="calendarWidget"/>
            </item>
            <item>
             <widget class="QTextEdit" name="menuTextEdit"/>
            </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
            <item>
             <widget class="QPushButton" name="btnVisualizarMenu">
                <property name="text">
                 <string>Visualizar Menú</string>
                </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnVolver">
                <property name="text">
                 <string>Volver</string>
                </property>
             </widget>
            </item>
         </layout>
        </item>
     </layout>
    </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
